doc/release/2021-08-releases.rst

```diff
@@ -49,28 +52,140 @@ Core
+    this transaction will be rollbacked if it does not have time to commit before this moment.
+
+-   Add new ``memtx_allocator`` option to ``box.cfg{}``.
+    It allows selecting the appropriate allocator for memtx tuples if necessary.
+    Possible values are ``system`` for malloc allocator and ``small`` for default small allocator.
+
+    Implement system allocator, based on malloc: slab allocator, used for tuples allocation,
+    has a particular disadvantage --- it tends to unresolvable fragmentation on specific workloads (size migration).
+    In this case, a user should be able to choose another allocator.
+    System allocator is based on malloc function but restricted by the same quota as a slab allocator.
+    The system allocator does not alloc all memory at the start. Instead, it allocates memory as needed,
+    checking that quota is not exceeded
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5419`).
+
+Replication
+~~~~~~~~~~~
+
+-   Introduce ``box.info.replication[n].downstream.lag`` field to monitor the state of replication.
+    This member represents a lag between a moment when the main node writes a certain transaction to its own WAL
+    and a moment it receives an ack for this transaction from a replica
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5447`).
+
+-   Introduce ``on_election`` triggers. The triggers may be registered via ``box.ctl.on_election()`` interface
+    and are run asynchronously each time ``box.info.election`` changes
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5819`).
+
 
 LuaJIT
 ~~~~~~
 
 -   Introduce support for ``LJ_DUALNUM`` mode in ``luajit-gdb.py``
-    (:tarantool-release:`2.8.2`, :tarantool-release:`2.7.3`, :tarantool-issue:`6224`).
+    (:tarantool-release:`2.8.2`, :tarantool-release:`2.7.3`,
+    :tarantool-release:`2.10.0-beta1`, :tarantool-issue:`6224`).
+
+-   Introduce preliminary support of GNU/Linux ARM64 and MacOS M1 (:tarantool-release:`2.10.0-beta1`).
+    In the scope of this activity, the following issues have been resolved:
+
+    *   Introduce support for a full 64-bit range of lightuserdata values (:tarantool-issue:`2712`)
+
+    *   Fixed memory remapping issue when the page left 47-bit segments
+
+    *   Fixed M1 architecture detection (:tarantool-issue:`6065`)
+
+    *   Fixed variadic arguments handling in FFI on M1 (:tarantool-issue:`6066`)
+
+    *   Fixed ``table.move`` misbehaviour when table reallocation occurs (:tarantool-issue:`6084`)
+
+    *   Fixed Lua stack inconsistency when ``xpcall`` is called with the invalid second argument on ARM64
+        (:tarantool-issue:`6093`)
+
+    *   Fixed ``BC_USETS`` bytecode semantics for closed upvalues and grey strings
+
+    *   Fixed side exit jump target patching considering the range values of the particular instruction
+        (:tarantool-issue:`6098`)
+
+    *   Fixed current Lua coroutine restoring on the exceptional path on ARM64 (:tarantool-issue:`6189`)
```

Comment:
```suggestion
    *   Fixed M1 architecture detection (:tarantool-issue:`6065`).

    *   Fixed variadic arguments handling in FFI on M1 (:tarantool-issue:`6066`).

    *   Fixed ``table.move`` misbehavior in case of table reallocation (:tarantool-issue:`6084`).

    *   Fixed Lua stack inconsistency when ``xpcall`` is called on ARM64 with the invalid second argument
        (:tarantool-issue:`6093`).

    *   Fixed ``BC_USETS`` bytecode semantics for closed upvalues and grey strings.

    *   Fixed side exit jump target patching considering the range values of the particular instruction
        (:tarantool-issue:`6098`).

    *   Fixed current Lua coroutine restoring on the exceptional path on ARM64 (:tarantool-issue:`6189`).
```
added periods
still have some questions :)